thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/validators/SubscriptionConfigValidator.java

```diff
@@ -90,10 +90,6 @@ public void validateConfig(DetectionAlertConfigDTO alertConfig) throws IllegalAr
    */
   @Override
   public void validateYaml(Map<String, Object> config) throws IllegalArgumentException {
-    // Make sure recipients are configured at the right place
-    Preconditions.checkArgument(!config.containsKey(PROP_RECIPIENTS),
```

Comment:
Are we removing this? Seems unrelated.

Reply:
That's already remove in another PR and merged. I need to rebase the changes.